api/proto/info/info.proto

```diff
@@ -504,9 +506,44 @@ message ZInfoNetworkInstance {
     ZInfoVpn vinfo = 31;
     ZInfoLisp linfo = 32;
   }
+  UsageInfo usage = 33;
```

Comment:
What does adding UsageInfo to the network instance have to do with volumes and contenttrees?

Are we adding UsageInfo to all the objects? And should that be part of this PR?

Reply:
removed it. We can add UsageInfo to other objects as part of separate PR.